src/coreclr/inc/eetwain.h

```diff
@@ -31,9 +31,7 @@
 #include "bitvector.h"
 #include "gcinfotypes.h"
 
```

Comment:
This won't work for regular CoreCLR. Switching regular CoreCLR to use `USE_GC_INFO_DECODER` would be a ton more work.

Reply:
Please forgive my ignorance. My pattern matching sometimes works like monkey typing on typewriter. I not fully understand boundaries, so I occasionally overstep. Do not want to do more work then necessary.